rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -128,6 +128,23 @@
             @QueryParam("index") @DefaultValue("-1") int index, @QueryParam("limit") @DefaultValue("-1") int limit)
             throws PermissionRestException, NotConnectedRestException;
 
+    /**
+     * Returns a list of jobs with each jobs is described using -
+     * its id - its owner - the JobInfo class
```

Comment:
The description is a bit strange, to check. 
And we probably should not mention `owner`.